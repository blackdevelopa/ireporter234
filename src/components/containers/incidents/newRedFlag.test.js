/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { shallow } from 'enzyme';
import NewRedflag from './NewRedFlag';

const props = {
  success: false,
  incident: {
    location: 'location',
    comment: 'comment',
  },
};

const initialState = {
  isLoading: false,
  success: false,
  redflagReducer: {},
  errors: {},
};

const mockStore = configureMockStore([thunk]);
const store = mockStore(initialState);

const connectedWrapper = shallow(
  <Provider store={store}>
    <Router>
      <NewRedflag {...props} />
    </Router>
  </Provider>
);

describe('New Redflag page', () => {
  it('should render without crashing', () => {
    expect(connectedWrapper).toMatchSnapshot();
  });
});
