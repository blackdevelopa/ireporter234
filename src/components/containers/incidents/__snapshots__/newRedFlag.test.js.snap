// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewRedflagPage /> should render without crashing 1`] = `ShallowWrapper {}`;
